### PR TITLE
[ARCTIC-116][AMS]   Support table operations tab

### PR DESCRIPTION
**Pull request by Aireed**
add operations tab to table detail tab list
link https://github.com/NetEase/arctic/issues/116

### CODE REVIEW

**Comment by hameizi** [comment]
there need add logic : order by commit desc 
most LGTM

**Comment by Aireed** [comment]
> there need add logic : order by commit desc most LGTM

done

File: ams/ams-server/src/test/java/com/netease/arctic/ams/server/controller/TableControllerTest.java [hunk 1]
```diff
@@ -275,6 +278,19 @@ public void testGetPartitionFileListInfo() throws Exception {
...
 
+  @Test
+  public void testGetOperations() throws Exception {
+    mockService(catalogName, database, table);
+    JavalinTest.test((app, client) -> {
+      app.get("/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/operations", TableController::getTableOperations);
+      String url = String.format("/tables/catalogs/%s/dbs/%s/tables/%s/operations", catalogName, database, table);
+      final okhttp3.Response resp = client.get(url, x -> {});
+      OkResponse result = JSONObject.parseObject(resp.body().string(), OkResponse.class);
+      LOG.info("xxx: {}", JSONObject.toJSONString(result));
```
Comment (SteNicholas) [inline ams/ams-server/src/test/java/com/netease/arctic/ams/server/controller/TableControllerTest.java:289]: Removes this line.

File: ams/ams-server/src/test/java/com/netease/arctic/ams/server/controller/TableControllerTest.java [hunk 2]
```diff
@@ -275,6 +278,19 @@ public void testGetPartitionFileListInfo() throws Exception {
...
+      assert result.getCode() == 200;
```
Comment (SteNicholas) [inline ams/ams-server/src/test/java/com/netease/arctic/ams/server/controller/TableControllerTest.java:290]: Please verify the operations of result.

**Comment by SteNicholas** [comment]
@Aireed, IMO, the title of the PR could update to `[ARCTIC-116][AMS] Support table operations tab`.

File: ams/ams-server/src/test/java/com/netease/arctic/ams/server/controller/TableControllerTest.java (hunk 2 above)
Comment (Aireed) [reply ams/ams-server/src/test/java/com/netease/arctic/ams/server/controller/TableControllerTest.java:291]: done.  add some pageing test.

**Comment by Aireed** [comment]
> @Aireed, IMO, the title of the PR could update to `[ARCTIC-116][AMS] Support table operations tab`.

Thanks for your advice!   i have updated the title!

**Review by SteNicholas — APPROVED** [review]
LGTM. cc @zhoujinsong

**Review by zhoujinsong — APPROVED** [review]

**Pull request merged**